pkg/trace/obfuscate/sql.go

```diff
@@ -250,13 +282,14 @@ type ObfuscatedQuery struct {
 func attemptObfuscation(tokenizer *SQLTokenizer) (*ObfuscatedQuery, error) {
 	filters := []tokenFilter{
 		&discardFilter{},
-		&replaceFilter{},
-		&groupingFilter{},
 	}
 	tableFinder := &tableFinderFilter{}
-	if config.HasFeature("table_names") {
+	if config.HasFeature("table_names") || config.HasFeature("normalize_sql_tables") {
 		filters = append(filters, tableFinder)
 	}
+	filters = append(filters, &replaceFilter{})
+	filters = append(filters, &groupingFilter{})
```

Comment:
```suggestion
	filters = append(filters, &replaceFilter{}, &groupingFilter{})
```
Should be fine to do this, I think.